docs/user_guide/config.md

```diff
@@ -226,9 +227,10 @@ Below is an example implementation of a reverse proxy, using a custom Docker Com
                   NB_API_ALLOWED_ORIGINS: ${NB_NAPI_ALLOWED_ORIGINS:-"*"}
                   NB_ENABLE_AUTH: ${NB_ENABLE_AUTH:-false}
                   NB_QUERY_CLIENT_ID: ${NB_QUERY_CLIENT_ID}
-                  VIRTUAL_HOST: myservice1.myinstitute.org 
-                  LETSENCRYPT_HOST: myservice1.myinstitute.org 
+                  VIRTUAL_HOST: myservice1.myinstitute.org
+                  VIRTUAL_PATH: ${NB_NAPI_BASE_PATH:-/}
```

Comment:
Question: did we confirm that this doesn't work with `NB_NAPI_BASE_PATH=""` (e.g., if one service is hosted on the subdomain root and others on subpaths, or when all services on on different subdomains?)

If so, we might also want to change the default/comments in the template.env to avoid confusion, e.g.
https://github.com/neurobagel/recipes/blob/58e5acc5c694777171bf773afd4fe26341ed530f/template.env#L59-L62